azblob/zt_url_service_test.go

```diff
@@ -9,6 +9,14 @@ import (
 	chk "gopkg.in/check.v1" // go get gopkg.in/check.v1
 )
 
+func (s *aztestsSuite) TestGetAccountInfo(c *chk.C) {
+	sa := getBSU()
+
+	// Ensure the call succeeded. Don't test for specific account properties because we can't/don't want to set account properties.
```

Comment:
>// Ensure the call succeeded. Don't test for specific account properties because we can't/don't want to set account properties. [](start = 1, length = 127)

perhaps validate that it's at least not empty